cmake_modules/ccache.cmake

```diff
@@ -0,0 +1,20 @@
+# Find programs and assign the boolean to the variables, find_program(variable binary)
+find_program(CCACHE ccache)
+find_program(GCC g++)
+find_program(CLANG clang++)
+
+# Some ccache logic
+if (CLANG)
+    set(CXX_COMPILER clang++)           # Set the compiler
+    set(CXX_COMPILER_PATH ${CLANG})     # Set the path of the compiler, for if ccache not is used
+elseif (GCC)
+    set(CXX_COMPILER g++)               # Set the compiler
+    set(CXX_COMPILER_PATH ${GCC})       # Set the path of the compiler, for if ccache not is used
+endif()
```

Comment:
I still dislike this a lot (if anything we should just do this in Roboteam_Suite once). If we put ccache in roboteam_suite once all of this will be fine and we don't need to duplicate it for every repo.

Reply:
Sure we can do that once we fix the structure?